code/modules/mob/living/silicon/ai/examine.dm

```diff
@@ -26,6 +26,9 @@
 			. += "The wireless networking light is blinking."
 		else if (!shunted && !client)
 			. += "[src]Core.exe has stopped responding! NTOS is searching for a solution to the problem..."
+	//SKYRAT EDIT ADDITION BEGIN - CUSTOMIZATION
+	. += get_silicon_flavortext()
+	//SKYRAT EDIT ADDITION END
```

Comment:
I'm not 100% on AI getting the general silicon flavortext, but I don't think it's a big deal in this specific case.

Reply:
It could potentially be its own thing, like pAI, but currently it's so that the hologram can get nicely examined.